test/unit/role_test.rb

```diff
@@ -133,6 +133,16 @@ class RoleTest < ActiveSupport::TestCase
       it { subject.must_include(first) }
       it { subject.wont_include(second) }
     end
+
+    context "when current user is admin for_current_user should return all roles" do
+      before do
+        User.current = users(:admin)
+      end
+      it "Admin user should query Role model with no restrictions" do
```

Comment:
I'll change the 'it' to 'test'. Not sure why rubocop is not complaning about the missing empty line between the setup block and this.  @alongoldboim don't worry I'll change that on merge.